environment-python-2.yml

```diff
@@ -70,3 +70,7 @@ dependencies:
   - thredds_crawler
   - xarray
   - xlrd
+  # R packages.
+  - r-irkernel
```

Comment:
Right now we have only the `R` kernel and its dependencies, and two oceanographic libraries. We should get suggestions from `R` users for more packages to add.